### PR TITLE
Nil object cannot be proto merged

### PR DESCRIPTION
**Pull request by raghav-g**
Since first parameter is nil, it wont get merged and will panic. Pointed it to a dummy object to avoid it.

---
This change is [Reviewable](https://reviewable.io/reviews/ttacon/libphonenumber/44)

### CODE REVIEW

**Comment by ttacon** [comment]
Cheers!

File: phonenumberutil.go [hunk 1]
```diff
@@ -3063,6 +3063,8 @@ func buildNationalNumberForParsing(
 func isNumberMatchWithNumbers(firstNumberIn, secondNumberIn *PhoneNumber) MatchType {
 	// Make copies of the phone number so that the numbers passed in are not edited.
 	var firstNumber, secondNumber *PhoneNumber
+	firstNumber = &PhoneNumber{}
+	secondNumber = &PhoneNumber{}
 	proto.Merge(firstNumber, firstNumberIn)
 	proto.Merge(secondNumber, secondNumberIn)
```
Comment (raghav-g) [inline phonenumberutil.go:3069]: test

File: phonenumberutil.go [hunk 2]
```diff
@@ -3063,6 +3063,8 @@ func buildNationalNumberForParsing(
 func isNumberMatchWithNumbers(firstNumberIn, secondNumberIn *PhoneNumber) MatchType {
 	// Make copies of the phone number so that the numbers passed in are not edited.
 	var firstNumber, secondNumber *PhoneNumber
+	firstNumber = &PhoneNumber{}
+	secondNumber = &PhoneNumber{}
```
Comment (raghav-g) [inline phonenumberutil.go:3067]: test

File: phonenumberutil.go [hunk 3]
```diff
@@ -3063,6 +3063,8 @@ func buildNationalNumberForParsing(
 func isNumberMatchWithNumbers(firstNumberIn, secondNumberIn *PhoneNumber) MatchType {
 	// Make copies of the phone number so that the numbers passed in are not edited.
 	var firstNumber, secondNumber *PhoneNumber
```
Comment (raghav-g) [inline phonenumberutil.go:3065]: test

Comment (raghav-g) [inline phonenumberutil.go:3065]: Test

File: phonenumberutil.go (hunk 2 above)
Comment (raghav-g) [reply phonenumberutil.go:3067]: Hello

**Pull request merged**